script.js

```diff
@@ -18,7 +18,7 @@ async function testRun() {
     await driver.sleep(2000);
     await driver.findElement(By.xpath("//input[@name='qty']")).sendKeys(2);
     await driver.sleep(2000);
-    await driver.findElement(By.xpath("//button[contains(.,'ADD TO CART')]")).click();
+    await driver.findElement(By.xpath("//button[contains(.,'CART')]")).click();
```

Comment:
instead of using await driver.findElement(By.xpath("//button[contains(.,'CART')]")).click();
use await driver.findElement(By.xpath("//button[contains(text(),'CART')]")).click();

Reply:
updated